statik/filesystem.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017 Pilosa Corp.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//go:generate statik -src=../webui -dest=..
+//
+// Package statik contains static assets for the Web UI. `go generate` or
+// `make generate-statik` will produce statik.go, which is ignored by git.
+
```

Comment:
I think this newline stops the above comment from being interpreted as package documentation